### PR TITLE
Updates/dashboard-removed-images

### PR DESCRIPTION
**Pull request by A-Munafarnisha**

### CODE REVIEW

File: src/pages/auth/ForgotPassword.vue [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-            <base-card :padding="true">
-                <v-card-text class="text-center">
-                    <v-avatar size="60" class="mb-4">
-                        <img src="@/assets/images/logo.svg" alt="" />
-                    </v-avatar>
+            <base-card :padding="currentRoute ==='forgot-password'">
+                <v-card-text class="text-center pt-0">
+                    <div class="d-flex align-center justify-center">
+                        <div class="w-50">
+                            <img src="@/assets/images/logos/PositiveMedia-logo.png"/>
```
Comment (arivazhaganark) [inline src/pages/auth/ForgotPassword.vue:8]: use `<v-img>` component

File: src/pages/auth/Login.vue [hunk 2]
```diff
@@ -4,13 +4,13 @@
       <v-col cols="12" md="7" class="ma-0 pa-0">
         <div class="sign-three relative">
           <div class="sign-three-content z-10">
-            <div class="section-left-content">
-              <h1 class="text-4xl font-weight-bold white--text">
+            <!-- <div class="section-left-content"> -->
```
Comment (arivazhaganark) [inline src/pages/auth/Login.vue:7]: Remove unwanted lines, instead of hide

File: src/pages/dashboard/index.vue [hunk 3]
```diff
@@ -1,32 +1,152 @@
...
+      lastName: { required },
+    },
   },
+  methods: {
+   formSubmit() {
+      this.$v.$touch();
+      if (this.$v.$invalid) {
+        return;
+      }
+      console.log('submit')
```
Comment (arivazhaganark) [inline src/pages/dashboard/index.vue:84]: remove console.log methods

File: src/plugins/vuetify.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import Vuetify from "vuetify/lib";
 import { themePreset } from "../../src/themeConfig";
 
 Vue.use(Vuetify);
-
+console.log(themePreset.theme)
```
Comment (arivazhaganark) [inline src/plugins/vuetify.js:8]: remove console.log methods

File: src/pages/dashboard/index.vue (hunk 3 above)
Comment (A-Munafarnisha) [reply src/pages/dashboard/index.vue:84]: removed

File: src/pages/auth/ForgotPassword.vue (hunk 1 above)
Comment (A-Munafarnisha) [reply src/pages/auth/ForgotPassword.vue:8]: changed

File: src/plugins/vuetify.js (hunk 4 above)
Comment (A-Munafarnisha) [reply src/plugins/vuetify.js:8]: removed

**Pull request merged**